### PR TITLE
Bump version and update dependencies

### PR DESCRIPTION
**Pull request by cmyr**
Fingers crossed, this will be the last pre-release.

I'd like to at least get #340 and probably #342 in before doing a release, but this gets the paperwork in order.

### CODE REVIEW

**Comment by cmyr** [comment]
Thinking about it, we're just about ready to do the actual 0.2.0 release; I don't anticipate any more breaking changes.

File: piet-cairo/Cargo.toml [hunk 1]
```diff
@@ -10,12 +10,12 @@ keywords = ["graphics", "2d"]
 categories = ["rendering::graphics-api"]
 
 [dependencies]
-piet = { version = "0.2.0-pre5", path = "../piet" }
+piet = { version = "0.2.0-pre6", path = "../piet" }
```
Comment (raphlinus) [inline piet-cairo/Cargo.toml:13]: Did you mean "=0.2.0-pre6" here? Same is true for piet dep in the other back-ends.

**Comment by raphlinus** [comment]
I also have a not-great feeling about web text (#311), as this is a regression from the previous release. But I'm not sure that should block an 0.2, I think it can be fixed in a point release.

**Comment by cmyr** [comment]
Okay, decided to just fix the web text, ended up not being much to it at all. Let's get that merged, and then I'll pull the trigger on this.

**Pull request merged**